.github/workflows/release.yml

```diff
@@ -52,10 +52,13 @@ jobs:
       - run: cd bin-ubuntu-22.04 ; zip -qq -r ../opentelemetry-dotnet-instrumentation-linux-glibc-x64.zip . * ; cd ..
       - run: cd bin-ubuntu-22.04-arm ; zip -qq -r ../opentelemetry-dotnet-instrumentation-linux-glibc-arm64.zip . * ; cd ..
       - run: cd bin-windows-2022 ; zip -qq -r ../opentelemetry-dotnet-instrumentation-windows.zip . * ; cd ..
+      - run: cd bin-windows-2025 ; zip -qq -r ../opentelemetry-dotnet-instrumentation-windows.zip . * ; cd ..
```

Comment:
is there a need for 2025 publish separately?

Reply:
> Open question on which changes should be additional vs. replacing 2022. Possibly a mix of both.

I'm not sure yet. I figured I'd start with just adding 2025, see what broke if anything, and the update anything based on whether the tests should upgrade from 2022 to 2025, or _add_ 2025.